opba-rest-api-shared/src/main/java/de/adorsys/opba/restapi/shared/service/FacadeResponseMapper.java

```diff
@@ -93,4 +102,33 @@
         result.getHeaders().forEach(builder::header);
         return builder;
     }
+
+    private ResponseEntity.BodyBuilder setCookieHeaders(FacadeResultRedirectable<?, ?> result, ResponseEntity.BodyBuilder builder) {
+        if (StringUtils.isBlank(result.getToken())) {
```

Comment:
Avoid using commons just for 1 function, Guava has Strings.isNullOrEmpty

Reply:
done